src/planwise/client/scenarios/edit.cljs

```diff
@@ -65,10 +73,9 @@
         [common2/text-field {:label "Original capacity"
                              :read-only true
                              :value initial-capacity}])
-      [common2/numeric-text-field {:type "number"
-                                   :label (if increase? "Extra capacity" "Capacity")
-                                   :on-change  #(dispatch [:scenarios/save-key  [:changeset-dialog :change :capacity] %])
-                                   :value (or (:capacity change) "")}]
+      [common2/numeric-field {:label (if increase? "Extra capacity" "Capacity")
+                              :on-change  #(dispatch [:scenarios/save-key  [:changeset-dialog :change :capacity] %])
+                              :value (or (:capacity change) "")}]
```

Comment:
I don't think you need the `or` , since you're converting the value inside the `numeric-field` component. Besides, it's a bit weird for the value of a numeric field to be a string, but that's semantic nitpicking :)